Tensile/Source/client/include/DataInitialization.hpp

```diff
@@ -286,6 +286,11 @@ namespace Tensile
                 }
             }
 
+            size_t workspaceSize() const
```

Comment:
suggest revising naming as getWorkspaceSize()

Reply:
I follow naming in Tesnile